packages/flutter/test/material/slider_test.dart

```diff
@@ -3507,6 +3507,54 @@ void main() {
     );
   }, variant: TargetPlatformVariant.desktop());
 
+  testWidgets('Value indicator disappears after adjusting the slider', (WidgetTester tester) async {
+    // This is a regression test for https://github.com/flutter/flutter/issues/123313.
+    const double currentValue = 0.5;
+    await tester.pumpWidget(MaterialApp(
+      home: Material(
```

Comment:
To make this test future-proof. Let's Material 3 color to check value indicator visibility.
We won't need to update this later. 

```suggestion
    // This is a regression test for https://github.com/flutter/flutter/issues/123313.
    final ThemeData theme = ThemeData(useMaterial3: true);
    const double currentValue = 0.5;
    await tester.pumpWidget(MaterialApp(
      theme: theme,
      home: Material(
```